### PR TITLE
[bump] client: 2.0.0-rc.5.0.5 => 2.0.0-rc.5.0.6 (patch)

### PR DESCRIPTION
**Pull request by jzaffiro**
Bumped client from 2.0.0-rc.5.0.5 to 2.0.0-rc.5.0.6.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -28252,7 +28252,7 @@ packages:
     resolution: {integrity: sha512-m7Lllj9n/S6sOkCkRftpM7L24uvmfXQFedlW/4hENcuJH1HHm9u5EgxZb9uVjQSCGrbBWBkOGgcTxNg36r6ywA==}
     dependencies:
       fast-glob: 3.3.2
-      minimatch: 9.0.3
```
Comment (yann-achard-MS) [inline pnpm-lock.yaml:28255]: These changes seem unrelated to the bump

Comment (jzaffiro) [reply pnpm-lock.yaml]: Reverted

**Review by yann-achard-MS — APPROVED** [review]
